tests/test-clients/CMakeLists.txt

```diff
@@ -19,3 +19,4 @@ add_subdirectory(c-syscalls)
 add_subdirectory(cpp-sleep)
 add_subdirectory(cpp-threadnames)
 add_subdirectory(cpp-parallel)
+add_subdirectory(cpp-std-symbols)
```

Comment:
I don't think this is required or helpful. The string-based testing should be enough.

Reply:
Removed.